index.html

```diff
@@ -0,0 +1,139 @@
+
+        <ul class="menu-list" id="hum-list">
+            <li onclick="openMenu()" class=""><a class="" href="./about.html">About</a></li>
+            <li onclick="openmenu()" class=""><a class="" href="#program"></a>Program</li>
+            <li>Join</li>
+            <li>Sponsor</li>
+            <li>News</li>
+            <li>CC Campaign</li>
+        </ul>
+    </div>
+    </header>
+    <section class="main-section">
+        <div class="section-head">
+        <h1 class="greeting">"Hello! Sharing World"</h1>
+        <h2 class="topic">Research Conference and Seminar at MIC <br> 2023</h2>
+    </div>
+        <p class="detail">The seminars contribute to professional development, dissemination of research findings, and the advancement of knowledge within the respective field..
+        </p>
+        <div class="main-bottom">
+        <h3 class="time-fram">  2023.8.8(THU) ~ 16(WED)</h3>
+    <p class="address">@Maryland college hall Addis Ababa,Ethiopia</p>
+</div>
+    </section>
+    <section class="program" id="pro">
+        <div class="m-p">
+        <h1 class="m-pro">Main Program</h1>
+        <hr class="uderline"></hr>
+    </div>
+    <div class="pro-list">
+        <div class="lecture">
+            <img src="asset/lecture icon.png" alt="">
+            <p class="p-list">Lecture</p>
+            <p class="p-det">Listen to the speakers from various university.</p>        
+        </div>
+        <div class="Exhibition">
+            <img src="asset/CC icon.png" alt="">
+            <p class="p-list">CC Exhibition</p>
+            <p class="p-det">Appreciate various findings.</p>
+        </div>
+        <div class="forum">
+            <img src="asset/forum icon.png" alt="">
+            <p class="p-list">Forum</p>
+            <p class="p-det">Have the time to share your thoughts and opinions with scholars.
+            </p>
+        </div>
+        <div class="workshop">
+            <img src="asset/workshop icon.png">
+            <p class="p-list">Workshop</p>
+            <p class="p-det">Try creating your own work using open source license rather than just watching at it.
+            </p>
+        </div>
+        <div class="ignite">
+            <img src="asset/CC ignite icon.png">
+            <p class="p-list">CC Ignite</p>
+            <p class="p-det">get opportunities to network with proffesors from different university .
+            </p>
+        </div>
+        </div>
+        <div class="join-glo">
+        <h2 class="join">Join MIC Seminar 2023</h2>
+    </div>
+    <p class="see-pro">SEE THE HOLE PROGRAM</p>
+    </section>
+
+    <div class="f-speakers">
+        <h1>Featured Speakers</h1>
+        <hr>
+    </div>
+    <section class="featured-speakers" id="featured-section">
+    </section>
+    
+    <footer class="mobile-footer">
+        <div class="foot">
+        <div class="part">
+        <h2>Partner</h2>
+        <hr>
+    </div>
+        <div class="partners">
+            <ul class="partners-list">
```

Comment:
- [ ] please like in the original version, kindly ensure this section articles are organized in two columns of three articles each(6 total). kindly take note of their padding and margins, and the title color is not red but `#EC5242.`
yours
![Screenshot (1291)](https://github.com/Surafels/Capstone_Project/assets/65298048/22959166-3c89-49e9-9ed9-2b5bee065e63)
behance
![Screenshot (1292)](https://github.com/Surafels/Capstone_Project/assets/65298048/eae37380-bef7-48b0-81d6-d53697df7c1e)